--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,7 +7,7 @@
 arch=('aarch64')
 url='https://github.com/Genymobile/scrcpy'
 license=('Apache')
-depends=('android-tools' 'ffmpeg' 'sdl2')
+depends=('ffmpeg' 'sdl2')
 makedepends=('meson')
 optdepends=('android-tools: required if adb is not already installed')
 source=("https://github.com/Genymobile/scrcpy/archive/v${pkgver}.tar.gz"
@@ -26,7 +26,6 @@
         --prefix /usr \
         --buildtype release \
         --strip \
-        -Db_lto=true \
         -Dprebuilt_server="../scrcpy-server-v${pkgver}" \
         build
     ninja -C build
